adobe/exception.hpp

```diff
@@ -0,0 +1,31 @@
+/*
+    Copyright 2013 Adobe
+    Distributed under the Boost Software License, Version 1.0.
+    (See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+*/
+/**************************************************************************************************/
+
+#ifndef ADOBE_EXCEPTION_HPP
+#define ADOBE_EXCEPTION_HPP
+
+#include <string>
+
+/**************************************************************************************************/
+
+namespace adobe {
+
+/**************************************************************************************************/
+
+/// invoke `std::terminate` with the given message.
```

Comment:
? You can't invoke `std::terminate` with a message, so you need to fix this.

Reply:
I updated the comment to be more clear.